src/test/java/org/example/hsql/MigrateTablesFromH2Test.java

```diff
@@ -0,0 +1,81 @@
+package org.example.hsql;
```

Comment:
I don't think I want to include a test in here because it's so H2-specific. I think it makes sense that a user of ml-migration-starter would eventually want to put together a test for the migration code they're building, but we'd need more generic support so the user doesn't have to adapt/remove a lot of H2-specific code. 

I'll merge this into dev so there's a record of it, but I don't think I'll keep the test stuff present. I will keep the switch to H2 and the Gradle tasks, because that's an improvement over relying on a MySQL instance for tire-kicking.